core/templates/dev/head/base.html

```diff
@@ -349,41 +349,8 @@
     {% include 'components/activity_tiles_infinity_grid_directive.html' %}
     {% include 'components/loading_dots_directive.html' %}
 
+    {% include 'common-js-files.html' %}
```

Comment:
For consistency: common_js_files.html.

Having said that -- what is the rationale again for pulling this out to a separate file? Does it buy us anything?